nixos/modules/services/hardware/bluetooth.nix

```diff
@@ -1,12 +1,34 @@
 { config, lib, pkgs, ... }:
-
-with lib;
-
 let
   cfg = config.hardware.bluetooth;
-  bluez-bluetooth = cfg.package;
+  package = getBin cfg.package;
```

Comment:
`getBin` is currently doing nothing (because bluez doesn't have multiple outputs), but if it were I think it could break the loading of systemd, udev and dbus files below. Or are these guaranteed to be in the `.bin` output?

Reply:
Fair point.